release/nightly_tests/multimodal_inference_benchmarks/audio_transcription/ray_data_main.py

```diff
@@ -0,0 +1,105 @@
+from __future__ import annotations
+
+import io
+import time
+import uuid
+
+import numpy as np
+import ray
+import torch
+import torchaudio
+import torchaudio.transforms as T
+from transformers import AutoModelForSpeechSeq2Seq, AutoProcessor
+
+
+TRANSCRIPTION_MODEL = "openai/whisper-tiny"
+NUM_GPUS = 8
+SAMPLING_RATE = 16000
+INPUT_PATH = "s3://anonymous@ray-example-data/common_voice_17/parquet/"
+OUTPUT_PATH = f"s3://ray-data-write-benchmark/{uuid.uuid4().hex}"
+BATCH_SIZE = 64
+
+ray.init()
+
+
+@ray.remote
+def warmup():
+    pass
+
+
+# NOTE: On a fresh Ray cluster, it can take a minute or longer to schedule the first
+#       task. To ensure benchmarks compare data processing speed and not cluster startup
+#       overhead, this code launches a several tasks as warmup.
+ray.get([warmup.remote() for _ in range(64)])
+
+
+def resample(item):
+    audio = item.pop("audio")
```

Comment:
# NOTE: Remove the `audio` column since we don't need it anymore. This is done by
        # the system automatically on Ray Data 2.51+ with the `with_column` API.